tests/test_writer.py

```diff
@@ -1405,9 +1406,8 @@ def test_late_iss1654():
 @pytest.mark.enable_socket()
 def test_iss1723():
     # test of an annotation(link) directly stored in the /Annots in the page
-    url = "https://github.com/py-pdf/pypdf/files/11015242/inputFile.pdf"
     name = "iss1723.pdf"
-    reader = PdfReader(BytesIO(get_data_from_url(url, name=name)))
+    reader = PdfReader(BytesIO(get_data_from_url(name=name)))
```

Comment:
The changes in this file should probably be part of the other PR only and not of this one.

Reply:
PR https://github.com/py-pdf/pypdf/pull/2445 is about the naming conflict with test_iss1767.

I can create a new PR about the test test_iss1723 retrieving its test file from the local fs.

Reply:
Shouldn't it be sufficient to just revert these changes and just merge #2445 beforehand?

Reply:
Sure, we can do that.

I made that change because I thought that was the standard way to call a test file according to https://github.com/py-pdf/pypdf/issues/2442#issuecomment-1929015264.

Reply:
Reverted.

Reply:
> Sure, we can do that.

I made that change because I thought that was the standard way to call a test file according to https://github.com/py-pdf/pypdf/issues/2442#issuecomment-1929015264.

This was from when we had not yet discovered that the duplicate filename is indeed the culprit here. There is no real standard at the moment, but as GitHub downloads work correctly, there is o no real need as well. Apart from this, the change there is unrelated to your actual fix.